docs/contributing.md

```diff
@@ -1,13 +1,48 @@
 # Contributing to Elpis Cloud
 
-Say hi
-
 ## Reporting an Issue
 
 ## Making feature requests
 
 ## Contributing to the Documentation
 
+To help flesh out our documentation at `docs.elpis.com`, you'll first need to install
+[mkdocs](https://www.mkdocs.org/getting-started/).
+
+The easiest way to do this is with `pip install mkdocs` (or `pip3` for mac users).
+
+After that, from the project root, run `mkdocs serve` to serve the local version
+of the documentation. Now any changes you make should be live at [localhost:8000](localhost:8000)
+
+### Adding new pages
+
+There are a few steps to adding a new page to the online documentation.
+
+- Create a new markdown file for the page you want to add, at `docs/your_page.md`
+- If you want this page to appear in the navigation menu, you will need to edit
+  `mkdocs.yml` to include the new page under the `nav` section:
+
+```
+...
+
+nav:
+- Home: index.md
+- About: about.md
+...
+- Your Page Title: your_page.md
+```
+
+Your should now see your page in the sidebar!
+
+### `mkdocs` development commands
+
+- `mkdocs new [dir-name]` - Create a new project.
```

Comment:
Is this particular command relevant, I assume we've done this initial setp?

Reply:
Removed in next revision.